sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeBlockSuite.scala

```diff
@@ -133,4 +133,55 @@ class CodeBlockSuite extends SparkFunSuite {
            |}""".stripMargin
     assert(aliasedCode.toString == expected.toString)
   }
+
+  test ("transform expr in nested blocks") {
+    val expr = JavaCode.expression("1 + 1", IntegerType)
+    val isNull = JavaCode.isNullVariable("expr1_isNull")
```

Comment:
not related to this PR, shall we remove `isNullVariable` and always use `expression(..., BooleanType)`? Or we can add a bunch of `intExpression(...)`, `booleanExpression(...)` etc.

Reply:
I think it is fine. I was thinking the name of `isNullVariable`  is a bit confusing at the first look actually.